### PR TITLE
Scenario for adding webhook(day2)

### PR DESCRIPTION
**Pull request by varshab1210**
**What type of PR is this?**

/kind enhancement

**What does this PR do / why we need it**:

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**: 

Fixes #?
https://github.com/redhat-developer/kam/issues/137

**How to test changes / Special notes to the reviewer**:
```make e2e``` should pass

### CODE REVIEW

File: test/e2e/kamsuite/kamsuite.go [hunk 1]
```diff
@@ -27,7 +33,7 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^directory "([^"]*)" should exist$`,
 		DirectoryShouldExist)
 
-	s.Step(`^gitops repository is created$`,
+	s.Step(`^"([^"]*)" repository is created$`,
```
Comment (wtam2018) [inline test/e2e/kamsuite/kamsuite.go:36]: I am not following what this step is.  May be you can add a comment.

Comment (varshab1210) [reply test/e2e/kamsuite/kamsuite.go:37]: Previously we did not have the need to create any repository other than gitops. But for day2 operation, we are also using a bus repository. So we pass the name of the repository from [basic.feature](https://github.com/redhat-developer/kam/pull/282/files#diff-25040733ed2d476fece186ec77f656c9b53ddb09e77955c4d3e1b917901c4011R29) file and it'll create the repository with passed value.

**Review by wtam2018 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**